packages/react-core/src/components/Switch/Switch.tsx

```diff
@@ -30,8 +32,6 @@ export interface SwitchProps
   isDisabled?: boolean;
   /** A callback for when the switch selection changes. (event, isChecked) => {} */
   onChange?: (event: React.FormEvent<HTMLInputElement>, checked: boolean) => void;
-  /** Adds accessible text to the switch, and should describe the isChecked="true" state. When label is defined, aria-label should be set to the text string that is visible when isChecked is true. */
-  'aria-label'?: string;
   /** Flag to reverse the layout of toggle and label (toggle on right). */
```

Comment:
can you update the text while in there. Or we can open a follow up issue.  For RTL purposes it should say "end" rather than right".  I would advocate for actually changing the name to better align with other components. Maybe `togglePosition`.  we can introduce a new prop post v5.  i can add new issue for that.